runtime/src/accounts_db.rs

```diff
@@ -987,6 +997,25 @@ impl AccountsDB {
         )
     }
 
+    fn hash_frozen_account_data(account: &Account) -> Hash {
+        let mut hasher = Hasher::default();
+        let mut buf = [0u8; 8];
+
+        hasher.hash(&account.data);
```

Comment:
Why is the normal hash account not good?

Reply:
Need to exclude the slot and lamports fields from the hash

Reply:
Also I want to include the account owner in the hash

Reply:
ok